container/ClearSky/src/main/java/io/egen/clearskyboot/constants/HTTPStatus.java

```diff
@@ -0,0 +1,43 @@
+package io.egen.clearskyboot.constants;
+
+public final class HTTPStatus {
```

Comment:
Not sure if you are still using it, but Spring already has a class/enum similar to it.